eclass/kernel-2.eclass

```diff
@@ -1579,13 +1560,12 @@ kernel-2_src_unpack() {
 # Apply any user patches
 
 kernel-2_src_prepare() {
-
 	debug-print "Applying any user patches"
 
 	# apply any user patches
 	case ${EAPI:-0} in
 		0|1|2|3|4|5) epatch_user ;;
-		6) eapply_user ;;
+		*) eapply_user ;;
```

Comment:
Let’s drop a bunch of ancient EAPIs given EROOT is only defined in 3+ anyway? Probably other uses too but can’t check yet